gnomad/utils/filtering.py

```diff
@@ -341,3 +341,17 @@ def filter_to_clinvar_pathogenic(
         t.count_rows() if isinstance(t, hl.MatrixTable) else t.count(),
     )
     return t
+
+
+def remove_fields_from_globals(global_field: List[str], fields_to_remove: List[str]):
+    """
+    Remove fields from the pre-defined global field variables.
+
```

Comment:
might be helpful to include an example of when we'd use this function:
```suggestion

Used when exporting data to VCF. 
For example, used to remove "decoy" from `REGION_FLAG_FIELDS` in v3.1 VCF export.
```

Reply:
I changed some of the other info in the docstring, let me know if you still think an example of when to use is needed

Reply:
looks good to me! I added a slight rewording of the description (seems clearer to me, but no strong feelings)